test/GitcoinGovernor.t.sol

```diff
@@ -619,18 +627,26 @@ contract NewGitcoinGovernorProposalTest is GitcoinGovernorProposalTestHelper {
   function assertEq(IGovernor.ProposalState _actual, IGovernor.ProposalState _expected) internal {
     assertEq(uint8(_actual), uint8(_expected));
   }
+}
+
+contract NewGitcoinGovernorProposalTest is
+  GitcoinGovernorProposalTestHelper,
+  GovernorBravoProposalHelper
+{
+  function setUp()
+    public
+    virtual
+    override(GitcoinGovernorProposalTestHelper, GitcoinGovernorTestHelper)
+  {
+    GitcoinGovernorProposalTestHelper.setUp();
+  }
 
   function testFuzz_NewGovernorCanReceiveNewProposal(uint256 _gtcAmount, address _gtcReceiver)
     public
   {
     _assumeReceiver(_gtcReceiver);
     _upgradeToBravoGovernor();
-    (uint256 _newProposalId,,,,) =
-      _submitTokenSendProposal(address(gtcToken), _gtcAmount, _gtcReceiver);
-
-    // Ensure proposal is in the expected state
-    IGovernor.ProposalState _state = governorBravo.state(_newProposalId);
-    assertEq(_state, IGovernor.ProposalState.Pending);
+    _submitTokenSendProposalToGovernorBravo(address(gtcToken), _gtcAmount, _gtcReceiver);
```

Comment:
Definitely don't feel strongly enough to hold up the PR or require a change, but just want to note that in general I don't love moving the assertions into the setup helpers. I think it's nice to have the nature of the assertions obvious in each test, even if it's duplicated. If I do extract an assertion I try to make that a separate method from setup so it's clear what's going on, so it would look something like.

```solidity
_submitTokenSendProposalToGovernorBravo(address(gtcToken), _gtcAmount, _gtcReceiver);
_assertPending(_newProposalId);
```

Again, no change required, just wanted to share a perspective on test clarity.

Reply:
Alternatively I sometimes like just renaming the function if it holds business-logic assertions, like `_submitTokenSendProposalToGovernorBravoAndAssertXYZ`. When the assertions are things like "making sure the storage overwrite was done successfully" then that would not be reflected in the var name / seems ok to hide in a helper method

Reply:
Yeah, I generally do prefer just having explicitness in tests, even if it means code duplication. But the duplication had been called out on a previous PR so I was trying to cut it down a bit here